ext/opcache/zend_accelerator_module.c

```diff
@@ -321,6 +321,7 @@ ZEND_INI_BEGIN()
 	STD_PHP_INI_BOOLEAN("opcache.huge_code_pages"             , "0"   , PHP_INI_SYSTEM, OnUpdateBool,      accel_directives.huge_code_pages,               zend_accel_globals, accel_globals)
 #endif
 	STD_PHP_INI_ENTRY("opcache.preload"                       , ""    , PHP_INI_SYSTEM, OnUpdateStringUnempty,    accel_directives.preload,                zend_accel_globals, accel_globals)
+	STD_PHP_INI_ENTRY("opcache.preload_user"                  , ""    , PHP_INI_SYSTEM, OnUpdateStringUnempty,    accel_directives.preload_user,           zend_accel_globals, accel_globals)
```

Comment:
If there is no Windows support, I think the ini directive should be `#ifndef ZEND_WIN32` only.

Reply:
I assume, Windows is not affected, but I'm not completely sure.

Reply:
I'm with @cmb69 here - it should be in an ``#ifndef ZEND_WIN32`` block (and the ``char *preload_user;`` too, of course).

Reply:
As I understand it, there is not need for `preload.user` on Windows, but @weltling may know better. :)